ThinkSharp.Licensing.DotNetStandard.Test/ThinkSharp.Licensing.DotNetStandard.Test.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net5.0;net462;net47;net471;net472;net48</TargetFrameworks>
```

Comment:
It's nice to test it for all frameworks. However the tests on my computer does not run directly because I would need to install .net framework developer pack for all frameworks defined above. What about targeting only "net452;netstandard2.0" (because that are the frameworks defined in Thinsharp.Licensing.csproj)?

Reply:
It is not possible to run tests on `netstandard2.0` or any other .Net standard. Unit tests (like executables) have to run on a specific framework, as far as I know. We could of coarse limit the number of frameworks to reduce the necessary developer packs.